handlers/admin/AdminDashboards.cfc

```diff
@@ -1,7 +1,8 @@
 component extends="preside.system.base.AdminHandler" {
 
-	property name="widgetService" inject="adminDashboardWidgetService";
-	property name="siteService"   inject="siteService";
+	property name="widgetService"  inject="adminDashboardWidgetService";
+	property name="siteService"    inject="siteService";
+	property name="featureService" inject="featureService";
 
```

Comment:
We should not use the same widget approach as frontend widgets here (which was not a great implementation, but also is a separate set of widgets and shouldn't be overlapped in this way).

Suggest the more preferable approach of a widget being able to supply an `isEnabled()` handler that can be fired when figuring out which widgets we have in the system.